test/halmos/HalmosTest.sol

```diff
@@ -47,6 +51,23 @@ contract HalmosTest is SymTest, Test {
         morpho.enableLltv(lltv);
         morpho.createMarket(marketParams);
         vm.stopPrank();
+
+        // for flashLoan
+        otherToken = new ERC20Mock();
+        flashBorrower = new FlashBorrowerMock(morpho);
+
+        // Enable symbolic storage
+        svm.enableSymbolicStorage(address(morpho));
+        svm.enableSymbolicStorage(address(loanToken));
+        svm.enableSymbolicStorage(address(collateralToken));
+        svm.enableSymbolicStorage(address(oracle));
+        svm.enableSymbolicStorage(address(irm));
+        svm.enableSymbolicStorage(address(otherToken));
+        svm.enableSymbolicStorage(address(flashBorrower));
+
+        // Set symbolic block number and timestamp
+        vm.roll(svm.createUint(64, "block.number"));
+        vm.warp(svm.createUint(64, "block.timestamp"));
```

Comment:
the block number and timestamp are no longer symbolic, but initialized as they are in Foundry. they need to be explicitly updated as in Foundry tests.

Reply:
Good to know ! Thanks !